flake.nix

```diff
@@ -145,8 +152,8 @@
         #     Everything that is not a derivation goes here
         #     Includes functions, attrsets and simple literal values shared across cells
         #     These are not exposed to the flake
-        #   hydraJobs :: installables
-        #     Jobsets for our Hydra CI
+        #   ciJobs :: installables
+        #     Jobsets for our Hydra and Cicero CI
```

Comment:
- We have five cell blocks -> we have eight cell blocks (line 140)
- Please add a 1-liner for the `pipelines.nix` and `actions.nix` cell blocks.